Code/Tools/ProjectManager/Source/ProjectsScreen.cpp

```diff
@@ -220,6 +225,26 @@ namespace O3DE::ProjectManager
         {
             QVector<ProjectInfo> projects = projectsResult.GetValue();
 
+            // additional
+            auto remoteProjectsResult = PythonBindingsInterface::Get()->GetProjectsForAllRepos();
+            if (remoteProjectsResult.IsSuccess() && !remoteProjectsResult.GetValue().isEmpty())
+            {
+                const QVector<ProjectInfo>& remoteProjects = remoteProjectsResult.GetValue();
+                for (const ProjectInfo& remoteProject : remoteProjects)
+                {
+                    auto foundProject = AZStd::ranges::find_if(projects,
+                        [&remoteProject](const ProjectInfo& value)
+                        {
+                            return remoteProject.m_id == value.m_id;
+                        });
+                    if (foundProject== projects.end())
```

Comment:
nitpick: missing whitespace before `==`
```suggestion
                    if (foundProject == projects.end())
```